src/guest.S

```diff
@@ -97,6 +97,15 @@ _run_guest:
     fscsr t0
     csrw sstatus, t1
 
+    /* Restore the guest vector state from GuestInfo. */
+    /* We do have not stack, and this function exits */
```

Comment:
Likely meant: "We don't have a stack..."

Reply:
fixed